connect/src/spirc.rs

```diff
@@ -800,6 +889,39 @@ impl SpircTask {
                     let now = self.now_ms();
                     self.device.set_is_active(true);
                     self.device.set_became_active_at(now);
+                    self.player.emit_session_connect_event(
+                        self.session.connection_id(),
+                        self.session.username(),
+                        true,
+                    );
+                    self.player.emit_session_client_changed_event(
+                        self.session.client_id(),
+                        self.session.client_name(),
+                        self.session.client_brand_name(),
+                        self.session.client_model_name(),
+                    );
+
+                    let auto_play = matches!(
+                        &*self
```

Comment:
Does it work without `&*` referencing-dereferencing?

Reply:
Nope.